kotlin-native/runtime/src/main/cpp/ObjCExportCoroutines.mm

```diff
@@ -48,24 +48,34 @@
   RETURN_RESULT_OF(Kotlin_ObjCExport_createContinuationArgumentImpl, completionHolder, exceptionTypes);
 }
 
+extern "C" OBJ_GETTER(Kotlin_ObjCExport_createUnitContinuationArgument, id completion, const TypeInfo** exceptionTypes) {
+  void (^typedCompletion)(NSError * _Nullable) = completion;
+  Completion twoArgumentCompletion = ^(_Nullable id result, NSError* _Nullable error) {
+    // typedCompletion will be retained by the containing block when the latter is copied
+    // from stack to heap via objc_retainBlock.
+    typedCompletion(error);
+  };
+
+  RETURN_RESULT_OF(Kotlin_ObjCExport_createContinuationArgument, twoArgumentCompletion, exceptionTypes);
+}
```

Comment:
One test failed (related to this function):
```sh
./gradlew testObjCExport -Ptest_flags="-Xbinary=memoryModel=experimental -Xcheck-state-at-external-calls"
```

to fix it, please apply the following patch:
```patch
--- a/kotlin-native/runtime/src/mm/cpp/CallsChecker.cpp
+++ b/kotlin-native/runtime/src/mm/cpp/CallsChecker.cpp
@@ -182,6 +182,7 @@ extern "C" const char* Kotlin_callsCheckerGoodFunctionNames[] = {
         "CFStringCreateCopy",
         "CFStringGetCharacters",
         "CFStringGetLength",
+        "_Block_object_assign",
         "class_addIvar",
         "class_addMethod",
         "class_addProtocol",
```

Brief explanation (totally unrelated to this MR): this patch tells the checker that it is ok to call this function in the "runnable" thread state. The latter happens because `objc_retainBlocks` is called in "runnable" state, which in turn calls copy helper, which calls `_Block_object_assign`.

"Runnable" thread state is a state in which thread is responsible for the interaction with GC. In particular, it shouldn't block. Also, it shouldn't call back to Kotlin. These conditions are met for `_Block_object_assign`.

Reply:
Got it, done.